CHANGELOG.md

```diff
@@ -10,6 +10,76 @@ Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfl
 -->
 <!--Start Of Releases (DO NOT DELETE THIS LINE)-->
 
+##[1.0.0] - 2022-06-16
```

Comment:
Does not render properly without space

```suggestion
## [1.0.0] - 2022-06-16
```

Could you also correct line 184 of util.py to avoid this happening in future? (can't suggest this as not near an edit for this PR).

Reply:
If we want that extra space, I think it's needed on lines 183, 186, 212, and 240 of the release script.